exercises/ruby-1-a/ruby_1_a.rb

```diff
@@ -1,3 +1,13 @@
-def word_with_most_vowels(sentence)
-  # Start your code here.
+class ShoppingCart
+  def initialize(items)
+    # Start your code here.
```

Comment:
We don't really need the "start" word here, I think.

Reply:
I don't know if this is possible in Ruby, but ideally we'd put some code here that would provide a nice error message when the student submits the code _as is_.

Here are two examples of how the C# and Go experiment solve this:

1. C#: https://github.com/exercism/research_experiment_1/blob/e60a36db7510030cb34e4b5e32e17a4baed802d5/exercises/csharp-1-a/Csharp1A.cs#L7
1. Go: https://github.com/exercism/research_experiment_1/blob/e60a36db7510030cb34e4b5e32e17a4baed802d5/exercises/go-1-a/go_1_a.go#L5

I don't know if  something like this is possible in Ruby?